static/js/src/ejsImageGallery.js

```diff
@@ -102,6 +129,25 @@ class ImageGallery {
             });
     }
 
+    _parseVideoId(url) {
+        let videoId = null
+
+        return new Promise((resolve, _) => {
+            const a = document.createElement('a')
+            a.href = url
+
+            if (a.hostname === 'youtu.be') {
+                videoId = a.pathname.substr(1, a.pathname.length)
+                resolve(videoId)
+
+                return
+            }
+
+            videoId = a.search.substr(3, a.search.length)
```

Comment:
This is not desirable. The actual videoId is a value of query-string param `v`. 
Also provide example format in comment.